futurex_openedx_extensions/dashboard/views.py

```diff
@@ -1186,3 +1186,20 @@ def get(self, request: Any, scope: str, slug: str) -> JsonResponse | Response:
             })
 
         return JsonResponse(ch.result_to_json(result), safe=False)
+
+
+@docs('ConfigEditableInfoView.get')
+class ConfigEditableInfoView(FXViewRoleInfoMixin, APIView):
+    """View to get the list of editable keys of the theme designer config"""
+    permission_classes = [FXHasTenantCourseAccess]
+    fx_view_name = ''
```

Comment:
give it a name please
```suggestion
    fx_view_name = 'fx_config_editable_fields'
```